app/src/main/res/layout/fragment_bio.xml

```diff
@@ -1,30 +1,110 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/scrollView2"
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:layout_margin="4dp"
-    android:fillViewport="true">
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/item_padding"
+    app:cardCornerRadius="@dimen/item_padding"
+    app:cardBackgroundColor="@color/dark_gray"
+    app:cardElevation="4dp">
 
-    <LinearLayout
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
         android:orientation="vertical">
 
-        <include
-            android:id="@+id/card_artist_info"
-            layout="@layout/card_artist_info"
+        <com.airbnb.lottie.LottieAnimationView
+            android:id="@+id/animationView"
+            android:layout_above="@+id/cl"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:lottie_autoPlay="true"
+            app:lottie_loop="true"
+            app:lottie_rawRes="@raw/group"/>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
A `ConstraintLayout` layout inside a `RelativeLayout` seems weird. Let's try to flatten the hirerachy by moving the `LottieAnimationView` inside ConstraintLayout and removing the RelativeLayout.

Reply:
Haha! I guess my habit of using RelativeLayout needs to be flushed XD
Sure I'll fix this soon.

Reply:
Done

Reply:
Why do we need another ConstraintLayout here? Is it possible to just keep the outermost ConstrainLayout and remove this one?

Reply:
I had grouped all the artist information that we get from our API and highlighted them uniquely to separate it from the wiki stuff.

Reply:
This second ConstraintLayout is like a container to fit all the artist information from our API in one place.

Reply:
Removed the extra ConstraintLayout